tests/test_exceptiongroup.py

```diff
@@ -0,0 +1,241 @@
+from sentry_sdk.utils import event_from_exception
+
+
+try:
+    # Python 3.11
+    from builtins import ExceptionGroup  # type: ignore
+except ImportError:
+    # Python 3.10 and below
+    ExceptionGroup = None
+
+
+def test_exceptiongroup():
+    exception_groups_available = ExceptionGroup is not None
```

Comment:
please use `pytest.mark.skipif` like this to skip instead of return in tests

https://github.com/getsentry/sentry-python/blob/9457282e02e2fec6d3d658f4e19c45a336e9e70b/tests/integrations/asyncio/test_asyncio_py3.py#L11-L13

Reply:
ok (there was some reason why I did it like that, but I cant remember. Lets try the decorator and see if the reason pops up again or not.)